mediacapture-streams/MediaStream-idl.https.html

```diff
@@ -19,33 +19,33 @@ <h1 class="instructions">Description</h1>
 <div id='log'></div>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
-<script src="/common/vendor-prefix.js" data-prefixed-objects='[{"ancestors":["navigator"], "name":"getUserMedia"},{"ancestors":["window"], "name":"MediaStream"}]'></script>
+<script src="/common/vendor-prefix.js" data-prefixed-objects='[{"ancestors":["window"], "name":"MediaStream"}]'></script>
```

Comment:
Is there a reason `MediaStream` remain prefixed? If not it can be removed in a separate PR.

Reply:
No reason I know of. `MediaStream` hasn't been prefixed in Firefox for ages.

However, `navigator.getUserMedia` still is. @Pehrsons how come we removed it here?

Reply:
I removed `navigator.getUserMedia` from this test in favor of `navigator.mediaDevices.getUserMedia`.

I can remove the MediaStream vendor-prefix here. Just an oversight by me, I grepped for vendor-prefixed getUserMedia and removed those.